packages/terra-list/src/ListItem.jsx

```diff
@@ -94,6 +100,10 @@ const ListItem = ({
     attrSpread['data-item-show-focus'] = 'true';
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
+  } else if (isRoleListbox) {
```

Comment:
What am I missing if we simply do ⬇️? I am assuming that the list item is "disabled" if it is not selectable. 

```

const attrSpread = {
   role: 'option',
   'aria-disabled': isSelectable,  
};

if (isSelectable) {
    attrSpread.onClick = ListUtils.wrappedOnClickForItem(onClick, onSelect, metaData);
    attrSpread.onKeyDown = ListUtils.wrappedOnKeyDownForItem(onKeyDown, onSelect, metaData);
    attrSpread.tabIndex = '0';
    attrSpread.role = 'option';
    attrSpread['aria-selected'] = isSelected;
    attrSpread['data-item-show-focus'] = 'true';
    attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
    attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
}
```

Reply:
That would mean setting role : `option` for all items which will give axe violations for all other cases as pointed out  by Dave in this [comment](https://github.com/cerner/terra-core/issues/2664#issuecomment-541790941)

Reply:
Fair enough. Then add the role iff `isSelectable` is true but I am trying to understand why we cannot leverage the value of `isSelectable` to set `aria-disabled`.

Reply:
I think @dkasper-was-taken 's suggestion on this issue was was to give consumers a way to indicate if a an item in a selectable list was disabled or not via an `isDisabled` prop. I'm not sure consumers would know when to use `isRoleListbox`:

```jsx
<List  role="listbox"/>
   // this item needs aria-disabled
   <Item isSelectable isDisabled />
   // this item needs aria-disabled
   <Item isSelectable={false}  isDisabled />
   <Item isSelectable isSelected />
</ List>
```

This would look something like:
```diff
const attrSpread = {};
  if (isSelectable) {
    attrSpread.onClick = ListUtils.wrappedOnClickForItem(onClick, onSelect, metaData);
    attrSpread.onKeyDown = ListUtils.wrappedOnKeyDownForItem(onKeyDown, onSelect, metaData);
    attrSpread.tabIndex = '0';
    attrSpread.role = 'option';
+   attrSpread['aria-disabled'] = isDisabled
    attrSpread['aria-selected'] = isSelected;
    attrSpread['data-item-show-focus'] = 'true';
    attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
    attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
  }
```

Reply:
That's the same line of thought that I had too and also simply assumed that if an item is not selectable (`isSelectable` is truthy), then `aria-disabled` should automatically be true. This way, we needn't really add a new prop.